File: Data Structures & Algorithms/is-subsequence/submission-3.cs
public class Solution {
    public bool IsSubsequence(string s, string t) {
        int m = s.Length;
        int n = t.Length;
        int i = 0;
        int j = 0;
        while(i<m && j<n)
        {
            if(s[i]==t[j])
            {
                i++;
            }
            j++;

        }
        return i==m;

    }
}

File: Data Structures & Algorithms/maximum-difference-between-even-and-odd-frequency-i/submission-3.py
class Solution:
    def maxDifference(self, s: str) -> int:
        freq={}
        for i in range(len(s)):
            if s[i] not in freq:
                freq[s[i]]=1
            else:
                freq[s[i]]+=1
        evenmin=sys.maxsize
        oddmax=0
        for val in freq.values():
            if val%2==0:
                evenmin=min(val,evenmin)
            else:
                oddmax=max(val,oddmax)
        return oddmax-evenmin
        
        
        